Makefile

```diff
@@ -1,3 +1,10 @@
+ifeq ($(OS),Windows_NT)
+SEPARATOR := ;
+else
+SEPARATOR := :
+endif
+
+
```

Comment:
I copied it over from Calypso. I remember that there were some issues with `make` commands on Windows. In the long run we should resolve those kind of issues by using `npm scripts` :D